src/main/java/net/dries007/tfc/world/classic/worldgen/vein/VeinRegistry.java

```diff
@@ -9,7 +9,6 @@
 import java.io.IOException;
```

Comment:
Currently, all of `config/tfc/` is reserved for ore veins, including subfolders, as it's scanned by `VeinRegistry`
Now keeping in mind all of this will be completely thrown out in 1.15, I still don't like this solution (restrict veins to top level folder, make damage resistance a subfolder). So instead, here's my proposal:
 - Move the tfc config parsing to a util class, `JsonConfigRegistry` which scans each file recursively in `config/tfc/`, and checks for a `#loader` field. Based on the value of this, either ask `VeinRegistry`, or `DamageResistanceRegistry` to handle the file. If the `#loader` field isn't present, use `VeinRegistry`.

Given the ability we have to choose the format of damage resistance entries, I think this is an acceptable solution for both keeping existing functionality and not too much effort / leaving this in a kludged up state. For instance, a damage entry would be as such:
```json
{
    "#loader": "tfc:damage_resistance",
    "minecraft:creeper": { "..." }
```

Then the changes you've made to both vein registration and parsing aren't nessecary.